### PR TITLE
feat: playback started event

### PR DESCRIPTION
**Pull request by dan-ziv**
### Description of the Changes

Add playback started event.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/player.js [hunk 1]
```diff
@@ -1647,6 +1654,18 @@ export default class Player extends FakeEventTarget {
...
   }
 
+  /**
+   * @function _onPlaying
+   * @return {void}
+   * @private
+   */
+  _onPlaying(): void {
+    if (!this._playbackStarted) {
+      this._playbackStarted = true;
```
Comment (yairans) [inline src/player.js]: need to reset on change media

**Comment by yairans** [comment]
add test

File: src/player.js (hunk 1 above)
Comment (dan-ziv) [reply src/player.js]: done

**Comment by dan-ziv** [comment]
@yairans  tests added

**Pull request merged**